src/components/views/dialogs/SlidingSyncOptionsDialog.tsx

```diff
@@ -27,47 +27,35 @@ import { useAsyncMemo } from "../../../hooks/useAsyncMemo";
 import { SettingLevel } from "../../../settings/SettingLevel";
 
 /**
- * Check that the server natively supports sliding sync.
+ * Check if the server declares native support for sliding sync via a proxy.
  * @param cli The MatrixClient of the logged in user.
  * @throws if the proxy server is unreachable or not configured to the given homeserver
  */
-async function syncHealthCheck(cli: MatrixClient): Promise<void> {
-    await cli.http.authedRequest(Method.Post, "/sync", undefined, undefined, {
-        localTimeoutMs: 10 * 1000, // 10s
-        prefix: "/_matrix/client/unstable/org.matrix.msc3575",
-    });
-    logger.info("server natively support sliding sync OK");
-}
-
-/**
- * Check that the proxy url is in fact a sliding sync proxy endpoint and it is up.
- * @param endpoint The proxy endpoint url
- * @param hsUrl The homeserver url of the logged in user.
- * @throws if the proxy server is unreachable or not configured to the given homeserver
- */
-async function proxyHealthCheck(endpoint: string, hsUrl?: string): Promise<void> {
+async function nativeSlidingSyncSupportCheck(cli: MatrixClient): Promise<void> {
```

Comment:
Using a proxy would mean that the server doesn't natively support sliding sync. This PR removes support for implementations that have sliding sync built in already like conduit. It also removes the option to enter your own sliding sync proxy url, but that wasn't very secure anyway, so that's fine

Reply:
>This PR removes support for implementations that have sliding sync built in already like conduit.
I had an inkling I'd break as much as I'd "fix". This was a bit of a sledgehammer 😄 

So I guess I should split them back out again? If there's no well-known, get it to do a check that the endpoint returns something for native support?